src/main/java/com/hedera/exchange/database/ExchangeRateAWSRD.java

```diff
@@ -79,27 +79,7 @@
 
 	private static final String LATEST_EXCHANGE_QUERY = "SELECT e1.expirationTime, e1.exchangeRateFile FROM exchange_rate AS e1 INNER JOIN (SELECT MAX(expirationTime) expirationTime FROM exchange_rate) AS e2 ON e1.expirationTime = e2.expirationTime LIMIT 1";
 
-	private static final String LATEST_ADDRESSBOOK_QUERY = "SELECT e1.expirationTime, e1.addressBook FROM address_book AS e1 INNER JOIN (SELECT MAX(expirationTime) expirationTime FROM address_book WHERE networkName = ?) AS e2 ON e1.expirationTime = e2.expirationTime LIMIT 1";
-
-	private static final String NEW_ADDRESSBOOK_QUERY = "SELECT\n" +
-			"        e1.e1_expirationtime,\n" +
-			"        e1.e1_addressbook \n" +
-			"    FROM\n" +
-			"        (SELECT\n" +
-			"            e1.expirationTime AS e1_expirationtime,\n" +
-			"            e1.addressBook AS e1_addressbook \n" +
-			"        FROM\n" +
-			"            address_book AS e1 LIMIT 1) AS e1 \n" +
-			"    INNER JOIN\n" +
-			"        (\n" +
-			"            SELECT\n" +
-			"                MAX(expirationTime) expirationTime \n" +
-			"            FROM\n" +
-			"                address_book \n" +
-			"            WHERE\n" +
-			"                address_book.networkName = ?\n" +
-			"        ) AS e2 \n" +
-			"            ON e1.e1_expirationtime = e2.expirationTime LIMIT 1";
+	private static final String LATEST_ADDRESS_BOOK_QUERY = "SELECT e1.expirationTime, e1.addressBook FROM address_book AS e1 INNER JOIN (SELECT MAX(expirationTime) expirationTime FROM address_book WHERE networkName = ?) AS e2 ON e1.expirationTime = e2.expirationTime LIMIT 1";
```

Comment:
I think it is preferable if the query can be read without scrolling right. Is there a particular reason this was moved to a one-liner?

Reply:
Getting formatting errors if I break it into separate lines. I tested the query which is split into multiple lines in dataGrip and got errors. Single line statement works best.